L1Trigger/L1TMuonBarrel/src/L1MuBMTEtaPatternLut.cc

```diff
@@ -101,11 +102,14 @@ cout<<"skata"<<endl;
   //                                       << file.getName() << endl; 
 
   // ignore comment lines 
-  file.ignoreLines(16);
  
+  int skip2=getIgnoredLines(file);
+  // cout<<" lines to skip2 "<<skip2<<endl;
+    file.ignoreLines(skip2);
+
```

Comment:
What do you do if there is a comment not at the beginning of the file? Can that happen?